.github/workflows/build.yml

```
@@ -6,8 +10,22 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 jobs:
+  analyze:
+    stage: build
```

actionlint (1.7.7)

unexpected key "stage" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)